job-descriptions/gtm-sales-operations.md

```diff
@@ -0,0 +1,55 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Go-to-market (Sales) Operations Analyst
```

Comment:
I would recommend going with sales or go-to-market (I prefer sales); another option would be Sales Strategy and Operations Analyst (sounds cooler 😄)